### ISSUE TITLE
[Request] EfficientSamplerCustom

### ISSUE THREAD

**Issue by dathide** [jags111/efficiency-nodes-comfyui#202]
Thank you for your work on these custom nodes. I have been using TCD sampler extensions a lot lately, and they require the use of SamplerCustom. I have also been trying to create XY plots to test different TCD settings, and I saw that your nodes can do this easily for the built-in samplers. I would appreciate a new node called something like EfficientSamplerCustom based on SamplerCustom that can use the XY plot script with sampler and sigmas. This is the TCD extension I am using: https://github.com/JettHu/ComfyUI-TCD

**Comment by jags111** [comment]
sure will work on this and update

**Comment by dathide** [comment]
Thanks for the quick response. Actually, it might be better to make a node called EfficientSamplerCustom (Advanced) based on SamplerCustom (Advanced) because that node accepts a custom guider input too. This will make it more compatible with other extensions that add new guiders and the built-in PerpNegGuider.

**Comment by patientx** [comment]
We need this for "align your steps" as well, am I correct ?

**Comment by dathide** [comment]
Yes, AYS is a scheduler node that provides sigmas and can only be used with the SamplerCustom nodes.
